tests/managed/overloads.cs

```diff
@@ -277,4 +277,37 @@ public static AllOperatorsWithFriendly Negate (AllOperatorsWithFriendly c1)
 			return new AllOperatorsWithFriendly (-c1.Value);
 		}	
 	}
+
+	public class EqualOverrides {
+		public int Value { get; private set; }
+
+		public EqualOverrides (int val)
+		{
+			Value = val;
+		}
+
+		public static bool operator == (EqualOverrides a, EqualOverrides b)
```

Comment:
that should be the operator case for `areEqual:`

what's missing the the friendly-named version, 
```
public static bool Equals (EqualOverrides a, EqualOverrides b) { ... }
```